packages/ocvImpact/R/utils_targeting.R

```diff
@@ -55,6 +55,9 @@ load_targets_by_country <- function(datapath, modelpath, country){
       message(paste0("Loading ", datapath, "/incidence/afro_2010-2016_lambda_5k_mean.tif"))
       afr <- raster::raster(paste0(datapath, "/incidence/afro_2010-2016_lambda_5k_mean.tif"))
     }
+    
+    bgd <- raster::raster(paste0(datapath, "/incidence/BGD_incid_5k_100.tif"))
```

Comment:
I'm a bit confused here... are we loading the bgd raster even if the modeled country is not bgd? If the modeled country is in africa, we only need to load the afr raster.

Reply:
Good point, I'll put this inside an if statement